tools/monorepo-utils/src/github/commands/replace-labels/index.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+
+/**
+ * Internal dependencies
+ */
+import { getIssuesByLabel, updateIssue } from '../../../core/github/repo';
+import { Logger } from '../../../core/logger';
+
+export const replaceLabelsCommand = new Command( 'replace-labels' )
+	.description( 'Replace labels of issues' )
+	.option(
+		'-o --owner <owner>',
+		'Repository owner. Default: woocommerce',
+		'woocommerce'
+	)
+	.option(
+		'-n --name <name>',
+		'Repository name. Default: woocommerce',
+		'woocommerce'
+	)
+	.option( '-l --label <label>', 'Label to filter by and replace' )
+	.option(
+		'-r --replacement-label <replacementLabel>',
+		'Label to use for replacement'
+	)
+	.option(
+		'--remove-if-starts-with <removeIfStartsWith>',
+		'Only remove the label if it already contains a label that starts with.'
+	)
+	.action( async ( options ) => {
+		const { label, owner, name, replacementLabel, removeIfStartsWith } =
+			options;
+
+		if ( ! label ) {
+			Logger.warn(
+				`No label supplied, going off the latest release version`
+			);
+			return;
+		}
+
+		Logger.startTask( `Querying by label: "${ label }"` );
+		const { results } = await getIssuesByLabel( { owner, name }, label );
+		Logger.endTask();
+
+		if ( results.length === 0 ) {
+			Logger.warn( `No issues found by label: "${ label }"` );
+			process.exit( 0 );
+		}
+
+		for ( const issue of results ) {
+			// Get labels by name only and filter out the existing label.
+			const labels = issue.labels
+				.map( ( l ) => ( typeof l === 'string' ? l : l.name ) )
+				.filter( ( l ) => l !== label );
```

Comment:
`getIssuesByLabel` above is not case sensitive, but this check is, so its accurately identifying `Hacktoberfest` with a `-l hacktoberfest` query but the old label isn't removed. We may as well go with the same behavior as Github on this one and make a lowercase comparison.

Reply:
Nice catch, fixed here: https://github.com/woocommerce/woocommerce/pull/50649/commits/8184dc462b3186244aeb50562f0005f359f53409